.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+image:
+  file: flow.Dockerfile
+tasks:
+  - name: Set up Flow auth from the provided FLOW_REFRESH_TOKEN, then pull the draft specified by FLOW_DRAFT_ID 
+    # We need the dot to allow the script to set environment variables
+    init: . ./init.sh
```

Comment:
supernit:  can we use `source ./init.sh` instead so it's more self-explanatory?

Reply:
Yep good idea, done!